### ISSUE TITLE
LightGBM is required but not mentioned in requirements.txt 

### ISSUE THREAD

**Issue by Yorko** [TeamHG-Memex/eli5#304]
Running on MacOS Sierra 10.12.6 and Python 3.6

after `pip install eli5` and imports get this:

OSError: dlopen(/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/lightgbm/lib_lightgbm.so, 6): Library not loaded: /usr/local/opt/libomp/lib/libomp.dylib
  Referenced from: /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/lightgbm/lib_lightgbm.so
  Reason: image not found

Actually, the same error as when importing LightGBM. So looks like a dependency not mentioned in requirements.txt.

**Comment by lopuhin** [comment]
@Yorko I think the reason is that if you have ``lightgbm`` installed, ``eli5`` would try to import it. If it's not installed, ``eli5`` would import fine, it's not a dependency.

Failure occurs only if ``lightgbm`` install is broken, so it does not look like a dependency not mentioned issue to me, but probably we can make it more clear that the error is happening in ``lightgbm``, not in ``eli5`` - likely we should ignore not only the ``ImportError`` but also ``OSError`` on import.

**Comment by Yorko** [comment]
You're right, worked fine with uninstalled LightGBM :)
Yes, indeed, maybe reporting LightGBM errors can be improved when `eli5` is imported.
